src/com/xilinx/rapidwright/util/BlackboxPopulator.java

```diff
@@ -100,9 +105,25 @@ public static boolean relocateModuleInsts(Design top, Module mod, String cellAnc
             mi.place(toSite);
         }
         System.out.println("\n");
+        top.getNetlist().consolidateAllToWorkLibrary();
         return true;
     }
 
+    /**
+     * Process the design after filing black boxes to match what Vivado expectation.
+     * @param top The design to process
+     * @param blackboxes The black boxes that was filled
+     */
+    public static void postProcessing(Design top, List<Pair<String, String>> blackboxes) {
+
+        top.getNetlist().resetParentNetMap();
+
+        for (Pair<String, String> toCellLoc : blackboxes) {
+            combinePIPonClockNets(top, toCellLoc.getFirst());
+        }
+
+        setPropertyValueInLateXDC(top, "HD.RECONFIGURABLE", "false");
```

Comment:
Can you add a comment as to the value of this line?

Reply:
This line disables DFX on the `top` cell. It seems like a Hardware Contract specific necessity. Perhaps it should go there?

Reply:
moved